### PR TITLE
feat(zk_toolbox): add custom token scenario to toolbox CI

### PR DESCRIPTION
**Pull request by manuelmauro**
## What ❔

Add test scenario for a custom token to toolbox CI

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.

### CODE REVIEW

File: core/tests/revert-test/tests/revert-and-restart.test.ts [hunk 1]
```diff
@@ -138,13 +134,10 @@ describe('Block reverting test', function () {
...
         tester = await Tester.init(ethClientWeb3Url, apiWeb3JsonRpcHttpUrl, baseTokenAddress);
         alice = tester.emptyWallet();
-        logs = fs.createWriteStream('revert.log', { flags: 'a' });
+        logs = fs.createWriteStream(`revert_${fileConfig.chain}.log`, { flags: 'a' });
     });
 
     step('run server and execute some transactions', async () => {
-        // Make sure server isn't running.
-        await killServerAndWaitForShutdown(tester, serverProcess).catch(ignoreError);
-
```
Comment (Deniallugo) [inline core/tests/revert-test/tests/revert-and-restart.test.ts:147]: don't forget to kill server here and let's make a test param which will be set to !fileConfig.loadFromFile || !process.env.NO_KILL

File: zk_toolbox/crates/zk_supervisor/src/commands/test/args/revert.rs [hunk 2]
```diff
@@ -12,4 +13,6 @@ pub struct RevertArgs {
     pub external_node: bool,
     #[clap(short, long, help = MSG_NO_DEPS_HELP)]
     pub no_deps: bool,
+    #[clap(short, long, help = MSG_NO_KILL_HELP)]
+    pub auto_kill: bool,
```
Comment (Deniallugo) [inline zk_toolbox/crates/zk_supervisor/src/commands/test/args/revert.rs:17]: no kill

File: zk_toolbox/crates/zk_supervisor/src/commands/test/args/recovery.rs [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+use crate::messages::{MSG_NO_DEPS_HELP, MSG_NO_KILL_HELP, MSG_TESTS_RECOVERY_SNAPSHOT_HELP};
 
 #[derive(Debug, Serialize, Deserialize, Parser)]
 pub struct RecoveryArgs {
     #[clap(short, long, help = MSG_TESTS_RECOVERY_SNAPSHOT_HELP)]
     pub snapshot: bool,
     #[clap(short, long, help = MSG_NO_DEPS_HELP)]
     pub no_deps: bool,
+    #[clap(short, long, help = MSG_NO_KILL_HELP)]
+    pub auto_kill: bool,
```
Comment (Deniallugo) [inline zk_toolbox/crates/zk_supervisor/src/commands/test/args/recovery.rs:13]: no kill

Comment (manuelmauro) [reply zk_toolbox/crates/zk_supervisor/src/commands/test/args/recovery.rs]: Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/1d5c81092c3afb4a58541f696356855d07f7a1c6

File: zk_toolbox/crates/zk_supervisor/src/commands/test/args/revert.rs (hunk 2 above)
Comment (manuelmauro) [reply zk_toolbox/crates/zk_supervisor/src/commands/test/args/revert.rs]: Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/1d5c81092c3afb4a58541f696356855d07f7a1c6

File: core/tests/revert-test/tests/revert-and-restart.test.ts (hunk 1 above)
Comment (manuelmauro) [reply core/tests/revert-test/tests/revert-and-restart.test.ts]: Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/aff37b4a4842927952987db81d5c2b4bb2ed78f2

File: core/tests/revert-test/tests/revert-and-restart.test.ts [hunk 4]
```diff
@@ -138,6 +139,9 @@ describe('Block reverting test', function () {
     });
 
     step('run server and execute some transactions', async () => {
+        // Make sure server isn't running.
+        await killServerAndWaitForShutdown(tester, serverProcess!).catch(ignoreError);
```
Comment (Deniallugo) [inline core/tests/revert-test/tests/revert-and-restart.test.ts:143]: It also should be under autokill

Comment (manuelmauro) [reply core/tests/revert-test/tests/revert-and-restart.test.ts]: Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/8a90f96dc6cc9ede1bf92b732f19f977b9ca99fc

**Review by Deniallugo — APPROVED** [review]

**Pull request merged**